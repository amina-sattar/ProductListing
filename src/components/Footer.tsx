import React from 'react';
import AppBar from '@mui/material/AppBar';
import { styled } from '@mui/material/styles';
import Toolbar from '@mui/material/Toolbar';
import Typography from '@mui/material/Typography';

const FooterBar = styled(AppBar)(({ theme }) => ({
  backgroundColor: '#fff',
  color: theme.palette.text.secondary,
  position: 'relative',
  top: 'auto',
  bottom: 0,
  zIndex: theme.zIndex.drawer + 1,
}));

const Footer = () => {
  return (
    <FooterBar>
      <Toolbar sx={{justifyContent:"center"}}>
        <Typography>
          Alle Preise sind in Euro (â‚¬) inkl. gesetzlicher Umsatzsteuer und
          Versandkosten.
        </Typography>
      </Toolbar>

    </FooterBar>
  );
};
export default Footer;
