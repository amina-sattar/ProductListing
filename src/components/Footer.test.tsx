import React from 'react';
import renderer from 'react-test-renderer';
import Footer  from './Footer';

test('test snapshot for Footer', () => {
  const component = renderer.create(
    <Footer/>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})