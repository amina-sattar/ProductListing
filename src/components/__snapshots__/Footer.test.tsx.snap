// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapshot for Footer 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-1ldzm7z-MuiPaper-root-MuiAppBar-root"
>
  <div
    className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-13vimk6-MuiToolbar-root"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      Alle Preise sind in Euro (â‚¬) inkl. gesetzlicher Umsatzsteuer und Versandkosten.
    </p>
  </div>
</header>
`;
